chemicaltoolbox/data_source/get_online_data/get_pdb.xml

```diff
@@ -0,0 +1,29 @@
+<tool id="get_pdb" name="Get PDB file" version="0.1.0">
```

Comment:
@simonbray a .shed.yml file was missing so this tool was not uploaded

Reply:
whoops :/
https://github.com/bgruening/galaxytools/pull/862